src/awesome/keygrabber.rs

```diff
@@ -44,8 +44,16 @@ pub fn keygrabber_handle(mods: Vec<Key>, sym: Key, state: wlr_key_state) -> rlua
              })
 }
 
+/// Check is the Lua callback function is set
+pub fn is_keygrabber_set(lua: &Lua) -> bool {
+    match lua.named_registry_value::<Function>(KEYGRABBER_CALLBACK) {
```

Comment:
Just use `is_ok()` here instead of the match

Reply:
hehe, no I mean use [the result is_ok function](https://doc.rust-lang.org/std/result/enum.Result.html#method.is_ok). `is_keygrabber_set` is a fine name